ang/civicase/CaseList.html

```diff
@@ -1,15 +1,14 @@
 <div crm-ui-debug="cases"></div>
 <div crm-ui-debug="viewingCase"></div>
 <div crm-ui-debug="viewingCaseTab"></div>
-<h1 crm-page-title>{{ pageTitle }}</h1>
 <div id="bootstrap-theme" class="clearfix civicase__container">
-
-  <div class="panel panel-default civicase-search-panel clearfix" ng-class="{'case-is-focused': caseIsFocused}" civicase-search="filters" hidden-filters="hiddenFilters" expanded="searchIsOpen" on-search="applyAdvSearch(selectedFilters)">
+  <h1 crm-page-title>Manage Cases</h1>  <!-- Added for SEO purpose (This will be hidden through css) -->
```

Comment:
I can see this element. Its not hidden yet.

Reply:
This will be styled (made hidden) in the next styling ticket C51-36.
Adding this in the PR description.